packages/lisk-p2p/src/peer.ts

```diff
@@ -13,45 +13,65 @@
  *
  */
 /* tslint:disable:interface-name */
+import { Logger, PeerState } from './p2p_types';
+
+// TODO: Use to create outbound socket connection inside peer object.
+// TODO: const socketClusterClient = require('socketcluster-client');
 
 export interface IPeerConfig {
 	readonly clock?: Date;
```

Comment:
do we use `clock`...?

Reply:
I added it because it was present in the core, but we can always remove or ignore it as its an optional field

Reply:
Since this is out of scope of this PR, lets fix it in another one